prime-router/src/main/resources/metadata/hl7_mapping/receivers/STLTs/MD/MD-receiver-transform.yml

```diff
@@ -0,0 +1,38 @@
+hl7Class: ca.uhn.hl7v2.model.v251.message.ORU_R01
+
+extends: classpath:/metadata/hl7_mapping/ORU_R01/ORU_R01-base.yml
+constants:
+  PID: '/PATIENT_RESULT/PATIENT/PID'
+
+elements:
+
+  - name: xtn-use-code-from-system
+    condition: '%resource.system = "email"'
+    value: [ '"NET"' ]
+    hl7Spec: [ '%{hl7TelecomPath}-2' ]
+
+  - name: md-message-encoding-characters
+    condition: true
```

Comment:
This transform is not needed. We generate ^~\& by default and doesn't need to be added. It's only needed when the value is different then ^~\&